versionupdate.ps1

```diff
@@ -73,4 +73,9 @@ foreach ($project in $csv) {
     }
 }
 
+if (-not $update -and $requireUpdate)
```

Comment:
I always forget what param to use, so instead of having to rerun with `-?`, I added a message to instruct.